adaptors/hybrisproximityadaptor/hybrisproximityadaptor.cpp

```diff
@@ -96,7 +95,7 @@ void HybrisProximityAdaptor::sendInitialData()
                ProximityData *d = buffer->nextSlot();
                d->timestamp_ = Utils::getTimeStamp();
                d->withinProximity_ = near;
-               d->value_ = near ? 10 : 0;
+               d->value_ = absinfo.value;
```

Comment:
Or rather it is the same place. Just dropped from the diff. So '_' was added back in, but is the ok for the logic to give far distance for near=true?